File: python/mario.py
from cs50 import get_int

while True:
    altura = get_int("Height: ")
    if altura <= 8 and altura > 0:
        break


for i in range(altura):

    for k in range(altura - i - 1):
        print(" ", end="")
    for h in range(i + 1):
        print("#", end="")

    print("  ", end="")

    for j in range(i + 1):
        print("#", end="")
    print("")

File: pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int altura;

    do
    {
        altura = get_int("Height:\n");
    }
    while (altura < 1 || altura > 8);

    for (int linha = 0; linha < altura; linha++)
    {
        for (int espaco = 0; espaco < altura - linha - 1; espaco++)
        {
            printf(" ");
        }
        for (int hash = 0; hash <= linha; hash++)
        {

            printf("#");
        }
        printf("  ");
        for (int hash = 0; hash <= linha; hash++)
        {

            printf("#");
        }
        printf("\n");
    }
}